src/transformer/utils/get_full_name.php

```diff
@@ -12,7 +12,7 @@ function get_full_name(\stdClass $user) {
     if ($has_first_name) {
         return $user->firstname;
     }
-    if ($has_lastname_name) {
+    if ($has_last_name) {
```

Comment:
Good find, amazed I managed to do that, must have been a dodgy name refactor.

Reply:
Yea, it looked a bit weird when I saw it, thought maybe it was a variable done later, but didn't see it, so I went ahead and changed it to what I thought it should be.

Reply:
Haha thanks